File: .NET Core/Exceptions/ServiceUnavailableBackendException.cs
using com.contidio.sdk.proto;
using System.Net;

namespace Contidio.Sdk.Exceptions
{
    public class ServiceUnavailableBackendException : BackendException
    {
        public ServiceUnavailableBackendException(string message)
            : base(BackendErrorCode.TEMPORARY_UNAVAILABLE, message, "ServiceUnavailableBackendException", HttpStatusCode.ServiceUnavailable)
        {
        }
    }
}

File: Java/src/main/java/com/contidio/sdk/exceptions/ServiceUnavailableBackendException.java
package com.contidio.sdk.exceptions;

import org.apache.commons.httpclient.HttpStatus;

import com.contidio.sdk.proto.MetadataProtocol.BackendErrorCode;

@SuppressWarnings("serial")
public class ServiceUnavailableBackendException extends BackendException {
	public ServiceUnavailableBackendException(String message) {
		super(BackendErrorCode.TEMPORARY_UNAVAILABLE, message, "ServiceUnavailableBackendException", 
				HttpStatus.SC_SERVICE_UNAVAILABLE);
	}	
}
